src/main/kotlin/io/github/inductiveautomation/kindling/zip/views/ToolView.kt

```diff
@@ -39,7 +38,7 @@ class ToolView(
     override fun customizePopupMenu(menu: JPopupMenu) = toolPanel.customizePopupMenu(menu)
 
     companion object {
-        fun maybeToolPath(path: Path): Boolean = Tool.find(path.toFile()) != null
+        fun maybeToolPath(path: Path): Boolean = Tool.find(path) != null
```

Comment:
Paths inside a Zip can't be converted to files, so this was failing, hence all the changes to filtering above.